postgresql.go

```diff
@@ -69,6 +69,10 @@ func (h *postgreSQL) init(db *sql.DB) error {
 	}
 
 	h.tablesHasIdentityColumn = make(map[string]bool)
+	err := h.buildTableHasIdentityColumn(db)
```

Comment:
please make it simliar to functions above, so:
* under grp.Go to get parallel execution
* pure function so `h.tablesHasIdentityColumn = h.buildTableHasIdentityColumn(db)` for simpler data flow